backend/src/activitypub/objects/index.ts

```diff
@@ -231,9 +236,12 @@ export async function sanitizeObjectProperties(properties: unknown): Promise<APO
 	if (!isAPObject(properties)) {
 		throw new Error('Invalid object properties. Expected an object but got ' + JSON.stringify(properties))
 	}
+
+	// prettier-ignore
 	const sanitized: APObject = {
```

Comment:
NOTE: Manually setting the type to `APObject` here is technically invalid because not all of the objects that are passed to the method have `id`, `type`, and `url` properties. As none of these properties are accessed in this method, the type-checking doesn't fail within the method. This explains why, despite not having an `id` or `type` property, we throw an error during mocking when we "overwrite" these properties by calling the `...properties` operation during variable assignment